portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,129 +21,219 @@
 import com.google.appengine.api.datastore.FetchOptions;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
 import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.appengine.api.datastore.QueryResultList;
 import com.google.gson.Gson;
 import com.google.sps.data.Comment;
 import com.google.sps.data.CommentsSend;
+
 import java.io.IOException;
+import java.lang.ClassCastException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.Collections;
 import java.util.ArrayList;
+import java.util.logging.Logger;
 import java.util.List;
 
 /* Servlet that stores and returns comments */
 @WebServlet("/comments")
 public class DataServlet extends HttpServlet {
+    // for debug and probably future uses
+    // final Logger log = Logger.getLogger(DataServlet.class.getName());
```

Comment:
Please do not add a commented code. You can keep it in some draft branch, but for the pull requests please try to present and merge the code as clean as possible.

Reply:
Okay, won't do it again